templates/vault.service.j2

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Vault server
+After=network.target
+
+[Service]
+ExecStart={{ vault_install_dir }}/vault server -config {{ vault_config_dir }}/vault.hcl
+StandardOutput=journal
```

Comment:
Would it be possible to send the logs to /var/log/vault.log like the Upstart script does? To have some consistency.